glyphs-reader/src/font.rs

```diff
@@ -1492,7 +1492,7 @@ fn parse_codepoint_str(s: &str, radix: u32) -> BTreeSet<u32> {
 fn default_master_idx(raw_font: &RawFont) -> usize {
     // Prefer an explicit origin
     // https://github.com/googlefonts/fontmake-rs/issues/44
-    raw_font
+    if let Some(Some(master_idx)) = raw_font
```

Comment:
instead of `Some(Some(` change the second `map` to `and_then`.

Reply:
done, thanks!  0834b8b9